lib/Zonemaster/Backend/DB/MySQL.pm

```diff
@@ -60,7 +60,11 @@ sub from_config {
         $host = "[$host]";
     }
 
-    my $data_source_name = "DBI:mysql:database=$database;host=$host;port=$port";
+    my $data_source_name = "DBI:MariaDB:database=$database;host=$host";
+
+    if ( $host ne 'localhost' and $host ne 'embedded' ) {
```

Comment:
The `embedded` value should be mentioned under MYSQL.host and MYSQL.port sections in Configuration.md.

Reply:
After all I am wondering if we really need to check for the `embedded` value. I added it because I found it was also a value used to bypass the port configuration. Maybe I can remove it.
I also don't know how an embedded server is usually used, maybe it requires to pass option with [_mariadb_embedded_options_](https://metacpan.org/pod/DBD::MariaDB#mariadb_embedded_options) which we don't support.

Reply:
Maybe we should raise an error if MYSQL.host = embedded? I haven't tried but I have a hunch embedded won't work for Backend.

Reply:
Good hunch, I get an error when setting MYSQL.host to `embedded`.
```
DBI connect(...) failed: Connection error: Embedded server is not supported
```
I'll remove this check since the DBI fails on its own. If someone needs it, we can easily re-add the condition.